### DISCUSSION TITLE
Regarding Contribution

### DISCUSSION THREAD

**Discussion by Tan141101** [alan-turing-institute/sktime#4211] [General]
Hello 
I would love to join your organization as a contributor regardless of being selected for gsoc or not as I want to learn more and more about the software field and contribute to your organization as I am really passionate about machine learning and am proficient with python and major deep learning frameworks as well. If you would please let me know in any way I can start learning by contributing.
Regards,
Tanmay

**Comment by fkiraly** [comment]
Welcome to `sktime`!

For getting started, have a look at our "getting started" guide:
https://github.com/sktime/sktime/issues/1147
Also feel free to join any of the community meet-ups on Fridays (discord)
